### PR TITLE
Add first_run_date to ping_info

### PR DESCRIPTION
**Pull request by mdboom**
This was a forgotten step of 1525045, which adds `first_run_date`.  It fell through the cracks because extra properties were still allowed on `ping_info` at that point, but going forward it will start complaining.

Checklist for reviewer:

- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [x] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

For glean changes:
- [x] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

File: validation/glean/baseline.2.core.pass.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "client_id": "6ff20eb7-e80d-4452-b45f-2ea7e63547aa",
     "seq": 1,
     "start_time": "2018-10-23 11:23:15-04:00",
-    "end_time": "2018-10-23 11:23:15-04:25"
+    "end_time": "2018-10-23 11:23:15-04:25",
+    "first_run_date": "2018-10-23 11:23:15-04:25"
```
Comment (fbertsch) [inline validation/glean/baseline.2.core.pass.json]: Are we doing true second resolution for `first_run_date`?

**Comment by mdboom** [comment]
Given discussion in Slack, I believe the plan is to merge this (and other open Glean PRs), move everything back to version 1, nuke existing data and update glean to start sending to version 1 again.

File: validation/glean/baseline.2.core.pass.json (hunk 1 above)
Comment (mdboom) [reply validation/glean/baseline.2.core.pass.json]: Nope.  Will fix in the validation content.

**Comment by mdboom** [comment]
@fbertsch : Given the plan, are you cool with this being merged?  (And I'll take care of moving everything back to version 1 after our open glean-related PRs are merged)?

**Pull request merged**